src/lib/pickles/step_verifier.ml

```diff
@@ -344,12 +344,12 @@ struct
         ( 'a
         , Inputs.Impl.Field.t Import.Scalar_challenge.t
         , _ )
-        Types.Step.Proof_state.Deferred_values.Plonk.Minimal.t )
+        Types.Wrap.Proof_state.Deferred_values.Plonk.Minimal.t )
```

Comment:
Oh, can you give more context about why this is now a wrap proof?

Reply:
It was always a wrap proof. The types aliased before; now they don't